sites/fast-website/package.json

```diff
@@ -0,0 +1,40 @@
+{
+  "name": "fast-website",
```

Comment:
Could you provide a better description in the package.json?  "FAST website is a culmination of marketing and documentation compiled for release to https://www.fast.design" for example?